### ISSUE TITLE
A magical grafana/influxdb support :-)

### ISSUE THREAD

**Issue by lucmichalski** [viasite/site-audit-seo#9]
Hi guys,

Hope you are all well !

Would it be possible to send your metrics into influxdb so we can display them in a grafana dashboard ?

Just check this example, and you ll get what I mean: https://github.com/FeliceGeracitano/webperf-dashboard

That would be awesome, as it would be easy to integrate the site-audit-seo-viewer into a dashboard.

I can help :-) 

Thanks for any insights or inputs on that.

Cheers,
Luc Michalski

**Comment by popstas** [comment]
Good idea! I did this in my another (closed) project. I am using InfluxDB with Grafana.

Maybe after web service release :)

**Comment by lucmichalski** [comment]
awesome :-)

**Comment by popstas** [comment]
Created MVP: points can be send to InfluxDB (too early to commit it).

We have to decide, how to group urls and values?

Show dashboard by url?
Create `$url` variable in Grafana?
Add domain to tags?

**Comment by lucmichalski** [comment]
Hi @popstas ,

Hope you are all well !

I think that they should be grouped by domain, and crawling session IDs if we want to compare results in time.

Like https://github.com/lucmichalski/garie-webperf-dashboard, maybe we should check how it is done.

I ll try to your mvp tomorrow, and thanks again for your active work :-) that s a pure pleasure to read you :-)

Cheers,
Luc Michalski

**Comment by popstas** [comment]
Thank you, I added `domain` and `url` as tags.

Will try webperf-dashboard.

**Comment by popstas** [comment]
See preview :)

![image](https://user-images.githubusercontent.com/3027126/103153486-abfa7200-47b2-11eb-8534-6b4bbfd54b97.png)
